elephant/utils.py

```diff
@@ -372,3 +394,54 @@ def get_opencl_capability():
         return True
     except ImportError:
         return False
+
+
+def trials_to_list_of_spiketrainlist(method):
+    """
+    Decorator to convert Trials object to a list of spiketrainlists before
+    calling the wrapped method.
+
+    Parameters
+    ----------
+    method: callable, The method to be decorated.
```

Comment:
Here we put the description in a new line

Reply:
Fixed.